dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -989,6 +996,8 @@ CLanStatus::InitializeNetTaskbarNotifications()
         pContext->hwndStatusDlg = hwndDlg;
         pItem->hwndDlg = hwndDlg;
 
+        pContext->nIDEvent = SetTimer(hwndDlg, NETTIMERID, 1000, NULL);
```

Comment:
Why is this moved here?
The WM_INITDIALOG used to call this?

Reply:
That's precisely the problem, timer has to be set when this function initialize the tray icon on session opening, and not on WM_INITDIALOG which is only called when the properties dialog is opened

Reply:
Except that that `WM_INITDIALOG` should be called before `CreateDialogParamW` returns.

Reply:
Looks like it's not the case. I'll dig on this